src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1960,14 +1982,19 @@ components:
       type: object
       properties:
         ancestor_identities:
+          description: (false by default) if set to true, ancestor addresses - parent_address, owner_address and global_address for entities are returned.
           type: boolean
         component_royalty_vault_balance:
+          description: (false by default) if set to true, royalty vault balance for component entities is returned.
           type: boolean
         package_royalty_vault_balance:
+          description: (false by default) if set to true, royalty vault balance for package entities is returned.
           type: boolean
         non_fungible_include_nfids:
+          description: (false by default) if set to true, first page of non fungible ids are returned for each non fungible resource, with cursor which can be later used at `/state/entity/page/non-fungible-vault/ids` endpoint.
           type: boolean
         explicit_metadata:
+          description: (empty by default) allows specifying explicitly metadata properties which should be returned in response, limited to max 20 items.
```

Comment:
We should not mention values that are a subject to runtime configuration, such as max items per page. Those are not hardcoded values.